app/controllers/volunteer_ops_controller.rb

```diff
@@ -13,18 +13,14 @@ def search
   end
 
   def index
-    @volunteer_ops = VolunteerOp.order_by_most_recent
-    @volunteer_ops = Feature.active?(:doit_volunteer_opportunities) ? @volunteer_ops : @volunteer_ops.local_only
+    @volunteer_ops = VolunteerOp.order_by_most_recent.send(restrict_by_feature_scope)
```

Comment:
@johnnymo87 I've extracted this to a private method displayed_volunteer_ops as you suggested.  Thanks :)